internal/handlers/handlers.go

```diff
@@ -1 +1,84 @@
 package handlers
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/Yandex-Practicum/go1fl-sprint6-final/internal/service"
+)
+
+const limit = 10 << 20 // 10 MB
+
+func Index(w http.ResponseWriter, r *http.Request) {
+	indexPage := "index.html"
+	f, err := os.ReadFile(indexPage)
+	if err != nil {
+		http.Error(w, "error reading index.html", http.StatusInternalServerError)
+		return
+	}
+	w.Header().Set("Content-Type", "text/html; charset=utf-8")
+	w.Header().Set("Content-Length", fmt.Sprint(len(f)))
+	w.WriteHeader(http.StatusOK)
+	w.Write(f)
+}
+
+func Upload(w http.ResponseWriter, r *http.Request) {
+	if r.Method != http.MethodPost {
+		http.Error(w, "not post method", http.StatusMethodNotAllowed)
+		return
+	}
+
+	r.ParseMultipartForm(limit)
+	file, handler, err := r.FormFile("myFile")
+	if err != nil {
+		http.Error(w, "error getting file", http.StatusBadRequest)
+		return
+	}
+	defer file.Close()
+
+	data := make([]byte, handler.Size)
+	_, err = file.Read(data)
+	if err != nil {
+		http.Error(w, "error reading file", http.StatusInternalServerError)
```

Comment:
Это тоже проблема клиента
```suggestion
		http.Error(w, "error reading file", http.StatusBadRequest)
```